src/data.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-mutable-exports */
+let currentUnit = 0; // 0 - Celsius 1 - Fahrenheit
+/* eslint-enable import/no-mutable-exports */
+/* eslint-enable prefer-const */
```

Comment:
Please avoid linter disabling as much as possible. Here, for example, instead of using a global variable to store the unit toggle value, try to find a way to use an object with an attribute where this value is stored.